docs/getting-started/quickstart.md

```diff
@@ -4,12 +4,24 @@ sidebar_position: 1
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import ThemedImage from '@theme/ThemedImage';
 
 # DBOS Quickstart
 
-This guide shows you how to deploy your first app to DBOS Cloud in less than 5 minutes. After that, it shows you how to develop DBOS apps locally.
+Let's create your first DBOS app! You start building an app by creating a special folder for it. From there, you can register with DBOS Cloud, provision a free Database Instance and deploy your app in a few minutes. You can also run DBOS apps on your computer.
 
-## Deploy Your First App to the Cloud
+<ThemedImage
+  alt="Docusaurus themed image"
+  sources={{
+    light: useBaseUrl('/img/quickstart-diagram.png'),
+    dark:  useBaseUrl('/img/quickstart-diagram-dark.png'),
+  }}
+/>
+
+In this guide, we start by deploying a sample "Hello" app to the cloud. After that, we provide instructions for how to run this app on your system.
+
+## Deploy your First App to the Cloud
 
 ### 1. Install Node.js version 20 or later
```

Comment:
Can we upgrade these instructions to node 22 (we require 20, but should be providing instructions for the latest stable version)

Reply:
Should be in place now